wasm-example/icu4x.mjs

```diff
@@ -0,0 +1,31 @@
+let icu4x;
```

Comment:
Nit: Please move these example files to a subdirectory somewhere under *ffi* so as not to clutter the top-level namespace.